### PR TITLE
use injector not mixed count for inactivity

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: f0ca412b29b3cf77fdd5b4d73db53a334a313abd available at: https://gist.github.com/17d902a0e85c3daaac45<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7076.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7076.dmg

File: assignment-client/src/audio/AudioMixerClientData.cpp [hunk 1]
```diff
@@ -157,12 +157,16 @@ void AudioMixerClientData::checkBuffersBeforeFrameSend() {
...
         SharedStreamPointer stream = it->second;
 
-        static const int INJECTOR_INACTIVITY_USECS = 5 * USECS_PER_SECOND;
+        if (stream->popFrames(1, true) > 0) {
+            stream->updateLastPopOutputLoudnessAndTrailingLoudness();
+        }
+
+        static const int INJECTOR_MAX_INACTIVE_BLOCKS = 500;
 
         // if we don't have new data for an injected stream in the last INJECTOR_INACTIVITY_MSECS then
```
Comment (zzmp) [inline assignment-client/src/audio/AudioMixerClientData.cpp]: This comment needs to be updated

File: assignment-client/src/audio/AudioMixerClientData.cpp [hunk 2]
```diff
@@ -170,12 +174,8 @@ void AudioMixerClientData::checkBuffersBeforeFrameSend() {
 
             // erase the stream to drop our ref to the shared pointer and remove it
             it = _audioStreams.erase(it);
-
+
```
Comment (zzmp) [inline assignment-client/src/audio/AudioMixerClientData.cpp]: white space

**Comment by hifi-gustavo** [comment]
Build console output for commit: 367e558b19963d9fb3b0646f30dab23966704e4d available at: https://gist.github.com/17d902a0e85c3daaac45<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7076.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7076.dmg

**Comment by OmegaHeron** [comment]
I built this and ran on Linux - it runs and audio works when I run an echo from server test - beyond that there's not much testing I can do.

**Pull request merged**